packages/fxa-auth-server/package.json

```diff
@@ -9,7 +9,7 @@
     "test": "test"
   },
   "scripts": {
-    "build": "tsc --build",
+    "build": "tsc --build && cp -R lib dist/fxa-auth-server && cp -R fxa-content-server-l10n dist/fxa-auth-server && cp -R config dist/fxa-auth-server",
```

Comment:
previously in the auth-server docker image we were copying the dist folder back into src. since we're phasing that out we need to copy the js into the compiled ts location. this is an unfortunate inconvenience of our mixed ts/js codebase.